src/App.tsx

```diff
@@ -52,24 +52,26 @@ class App extends React.Component<IAppProps> {
         const errorMessage = showError ? this.props.appError.message : "";
 
         return (
-            <Router>
-                <div className="app-shell">
-                    <Alert title={errorTitle}
-                        message={errorMessage}
-                        closeButtonColor="info"
-                        show={showError}
-                        onClose={this.props.actions.clearError}
-                    />
-                    <ErrorBoundary>
-                        <Navbar />
-                        <div className="app-main">
-                            <Sidebar project={this.props.currentProject} />
-                            <MainContentRouter />
+            <Fragment>
+                <Alert title={errorTitle}
+                    message={errorMessage}
+                    closeButtonColor="info"
+                    show={showError}
+                    onClose={this.props.actions.clearError}
+                />
+                <ErrorBoundary>
+                    <Router>
```

Comment:
Why are we introducing another `Router`? We are already using `Router` inside of `<MainContentRouter />`

Reply:
I'm not sure why it's there, but it's been there for a long time, wasn't introduced as part of this change.

Reply:
I don't see Router being use in MainContentRouter, is `<Route`> the same as `<Router>`

Reply:
You guys are right - sorry 👍